code/datums/datumvars.dm

```diff
@@ -905,7 +905,7 @@ client
 			if("brute")	L.adjustBruteLoss(amount)
 			if("fire")	L.adjustFireLoss(amount)
 			if("toxin")	L.adjustToxLoss(amount)
-			if("oxygen")L.adjustOxyLoss(amount)
+			if(OXYGEN)L.adjustOxyLoss(amount)

```

Comment:
inappropriate replacement, the damage type is desired here (ie someone needs to refactor this in the future, but it's unrelated to this PR)